src/bouncer/bouncer.service.ts

```diff
@@ -1,9 +1,12 @@
 import { Injectable, Logger } from '@nestjs/common'
 import { InjectBot } from 'nestjs-telegraf'
 import { DatabaseService } from 'src/database/database.service'
-import { Telegraf } from 'telegraf'
+import { Telegraf, TelegramError } from 'telegraf'
 import { Chat, User } from 'telegraf/typings/core/types/typegram'
 
+const OVERRIDE_DELAY = 60 * 1000 // 60 seconds
```

Comment:
Hmm. Vercel limits execution time to 10s. Could we have a shorter time window, like 5s..?

Reply:
Changed in https://github.com/opengovsg/telegovsg-bouncer/pull/1/commits/4cc0ba75ec4b957bb6dd8495ce853103ff5c54b1!